src/controllers/v1/entityTypes.js

```diff
@@ -67,9 +67,6 @@ module.exports = class EntityTypes extends Abstract {
 				query['tenantId'] = req.userDetails.tenantAndOrgInfo
 					? req.userDetails.tenantAndOrgInfo.tenantId
 					: req.userDetails.userInformation.tenantId
-				query['orgIds'] = req.userDetails.tenantAndOrgInfo
-					? { $in: req.userDetails.tenantAndOrgInfo.orgIds }
-					: { $in: [req.userDetails.userInformation.organizationId] }
```

Comment:
@borkarsaish65  line no 77 we are using orgIds , do we need to change that to orgId ?